src/components/AvailableConnections.js

```diff
@@ -21,7 +21,13 @@ export default function AvailableConnections() {
   const sourceTarget = useSelector(state => state.targets.source);
   const availableTargets = useSelector(state => state.targets.available);
   const apiOperationErrors = useSelector(state => state.resources.apiOperationErrors);
-
+  const collection_error = apiOperationErrors.find(
+    element => element.action === actionCreators.resources.loadFromSourceTarget.toString());
+  
+  let tokenError;
+  if (collection_error) {
+    tokenError = collection_error.data.includes('Token')
```

Comment:
Is there a status code we can check instead of a string?

Reply:
Yeah